src/utils/utils.ts

```diff
@@ -376,3 +376,15 @@ export const MESSAGE_ACTIONS = {
 
 export const makeImageCompatibleUrl = (url: string) =>
   (url.indexOf('//') === 0 ? `https:${url}` : url).trim();
+
+/** adopted from https://github.com/ai/nanoid/blob/master/non-secure/index.js */
+const alphabet =
+  'ModuleSymbhasOwnPr0123456789ABCDEFGHNRVfgctiUvzKqYTJkLxpZXIjQW';
+export const randomId = () => {
```

Comment:
this is in v2:
```js
export const generateRandomId = (a = ''): string =>
  a
    ? /* eslint-disable no-bitwise */
      ((Number(a) ^ (Math.random() * 16)) >> (Number(a) / 4)).toString(16)
    : `${1e7}-${1e3}-${4e3}-${8e3}-${1e11}`.replace(/[018]/g, generateRandomId);
```

Reply:
Mehh its fine!! Let's keep that in v2 and current randomId in v1. This one is copied from stream-chat-js